src/main/webapp/src/app/app.module.ts

```diff
@@ -86,15 +87,16 @@ export function initGapi(gapiSession: GapiSession) {
           },
         ],
       } as SocialAuthServiceConfig,
-    },
+    } /*
     {
       provide: APP_INITIALIZER,
       useFactory: initGapi,
       deps: [GapiSession],
       multi: true,
-    },
+    },*/,
```

Comment:
Just a quick question: what was the error when this was not commented?

Reply:
It was initializing OAuth twice. Your GapiSession once and AngularSocialLogin once. But feel free, in your dev branch, to do whatever you want to fix this as long as you don't break the public interface for `OAuthService`